addons/event_product/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Add the part of odoo licence comment in all python file that you modify/add in this PR